### ISSUE TITLE
Feature: Include contact names in JSON message export

### ISSUE THREAD

**Issue by ayancey** [tbvdm/sigtop#50]
When you run `sigtop export-messages`, it returns the sender's display name if the format is `text` or `text-short`. Because `json` output returns directly from the database, there's no display name included.

An option like `json-names` might be nice which joins sourceServiceId from messages table to serviceId in the conversations table.

Thanks!

**Comment by tbvdm** [comment]
Good point. Another possibility is to export the `json` column from the `conversations` table into a separate `conversations.json` file. Would that work for you?

**Comment by ayancey** [comment]
> Good point. Another possibility is to export the `json` column from the `conversations` table into a separate `conversations.json` file. Would that work for you?

That's a good idea. I'll take a whack at a PR, but I know next to nothing about Go. 😅

**Comment by aloisdg** [comment]
@ayancey abny update on the whack PR? I am not a Go dev either but I am willing to help
